tutorials/smart-contracts/deploy-erc20.md

```diff
@@ -32,7 +32,7 @@ To create the ERC-20 contract, you can follow the steps below:
 3. Now, paste the following ERC-20 contract code into the editor
 
     ```solidity title="MyToken.sol"
-    --8<-- 'code/tutorials/smart-contracts/deploy-erc20/MyToken.sol'
+    --8<-- 'https://raw.githubusercontent.com/polkadot-developers/polkavm-hardhat-examples/refs/tags/v0.0.1/erc20-hardhat/contracts/MyToken.sol'
```

Comment:
I'm sure you already have a plan, but any reason why we're not putting this in variables.yml?

Reply:
We're currently using releases rather than plain tags in our GitHub Actions workflow. To prevent breaking the CI, I'll probably use a different variable name for this case. I might use `tag` instead of `version`